### PR TITLE
Replace `@keywords internal` by `@noRd`

### LINKED ISSUE #557: Rd files for internal functions?
I noticed that there are Rd files that are not displayed on the website. I think these documents need to be improved and put on the website.

### PR DESCRIPTION
**Pull request by etiennebacher**
MERGE THIS AFTER #603 

---

Close #557

This doesn't change anything for the website since those were not displayed anyway. It only makes the package a bit cleaner (and lighter)

### CODE REVIEW

File: R/after-wrappers.R [hunk 1]
```diff
@@ -247,7 +247,7 @@ pl$show_all_public_methods = function(class_names = NULL) {
 #' @return an element from the public namespace `pl` polars. Likely a function or an RPolarsDataType
 #' @export
 #' @noRd
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/after-wrappers.R:250]: `@noRd` already exists in the line above.

**Review by eitsupi — CHANGES_REQUESTED** [review]
Thank you for working on this.
However, I think that a simple replacement may not enough.

File: R/after-wrappers.R (hunk 1 above)
Comment (etiennebacher) [reply R/after-wrappers.R:250]: It's annoying to go through all files to check this and it doesn't warn or error so let's just remove the duplicate when we come across one

**Comment by etiennebacher** [comment]
> However, I think that a simple replacement may not enough.

Why?

File: R/group_by.R [hunk 2]
```diff
@@ -32,14 +32,14 @@ RPolarsGroupBy = new.env(parent = emptyenv())
...
-#' @keywords internal
+#' @noRd
 .DollarNames.RPolarsGroupBy = function(x, pattern = "") {
   paste0(ls(RPolarsGroupBy, pattern = pattern), "()")
 }
 
 
 #' The internal GroupBy constructor
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/group_by.R:42]:
```suggestion
```

File: R/polars-package.R [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/polars-package.R:1]:
```suggestion
#' @keywords internal
```

File: R/s3_methods.R [hunk 4]
```diff
@@ -400,7 +400,7 @@ print.RPolarsSeries = function(x, ...) {
 #' @export
 #' @noRd
 #' @inherit .DollarNames.RPolarsDataFrame return
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/s3_methods.R:403]:
```suggestion
```

File: R/sql.R [hunk 5]
```diff
@@ -21,7 +21,7 @@ NULL
 #' @export
 #' @noRd
 #' @inherit .DollarNames.RPolarsDataFrame return
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/sql.R:24]:
```suggestion
```

File: R/utils.R [hunk 6]
```diff
@@ -83,7 +83,7 @@ verify_method_call = function(Class_env, Method_name, call = sys.call(1L), class
 #' list2 - one day like rlang
 #' list2 placeholder for future rust-impl
 #' @noRd
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:86]:
```suggestion
```

File: R/utils.R [hunk 7]
```diff
@@ -150,7 +150,7 @@ unpack_bool_expr = function(..., .msg = NULL) {
 #' @details Lifecycle: perhaps replace with something written in rust to speed up a bit
 #'
 #' @return any return given first true bool statement otherwise value of or_else
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:153]:
```suggestion
```

File: R/utils.R [hunk 8]
```diff
@@ -243,7 +243,7 @@ l_to_vdf = function(l) {
 #'
 #' @param env an R environment.
 #' @return shallow clone of R environment
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:246]:
```suggestion
```

File: R/utils.R [hunk 9]
```diff
@@ -361,7 +361,7 @@ construct_DataTypeVector = function(l) {
 #' @param pattern string passed to ls(pattern) to subset methods by pattern
 #' @details used internally for auto completion in .DollarNames methods
 #' @return method usages
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:364]:
```suggestion
```

File: R/utils.R [hunk 10]
```diff
@@ -505,7 +505,7 @@ restruct_list = function(l) {
 #' All R functions coined 'macro_'-functions use eval(parse()) but only at package build time
 #' to solve some tricky self-referential problem. If possible to deprecate a macro in a clean way
 #' , go ahead.
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:508]:
```suggestion
```

File: R/utils.R [hunk 11]
```diff
@@ -574,7 +574,7 @@ macro_new_subnamespace = function(class_pattern, subclass_env = NULL, remove_f =
 #'
 #' @param x object to view.
 #' @param collapse word to glue possible multilines with
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:577]:
```suggestion
```

File: R/utils.R [hunk 12]
```diff
@@ -592,7 +592,7 @@ str_string = function(x, collapse = " ") {
 #' @param allow_null bool, if TRUE accept NULL
 #'
 #' @return a result object, with either a valid string or an Err
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:595]:
```suggestion
```

**Review by eitsupi — CHANGES_REQUESTED** [review]

**Comment by etiennebacher** [comment]
Thanks for making the changes @eitsupi

File: R/after-wrappers.R [hunk 13]
```diff
@@ -16,7 +16,7 @@ build_debug_print = FALSE
 #' @param env environment object output from extendr-wrappers.R classes
 #' @param class_name optional class string, only used for debug printing
 #' Default NULL, will infer class_name automatically
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/after-wrappers.R:19]:
```suggestion
```

File: R/datatype.R [hunk 14]
```diff
@@ -22,7 +22,7 @@ pl$is_schema = is_schema
 #' mean undefined.
 #' @return bool
 #' @format function
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/datatype.R:25]:
```suggestion
```

File: R/error__trait.R [hunk 15]
```diff
@@ -8,7 +8,7 @@
 #' @details
 #' Additional details...
 #'
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/error__trait.R:11]:
```suggestion
```

File: R/error_conversion.R [hunk 16]
```diff
@@ -15,7 +15,7 @@
 #' error into an R error condition. These s3 methods can be implemented for any future error type.
 #'
 #' @return the ok-element of list , or a error will be thrown
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/error_conversion.R:18]:
```suggestion
```

File: R/error_conversion.R [hunk 17]
```diff
@@ -103,7 +101,7 @@ result = function(expr, msg = NULL) {
 #' @description use sparingly internally for speed optimization where the error is not important.
 #' @noRd
 #' @param expr code to capture any error from and wrap as Result
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/error_conversion.R:104]:
```suggestion
```

File: R/expr__expr.R [hunk 18]
```diff
@@ -129,7 +129,7 @@ wrap_e_result = function(e, str_to_lit = TRUE, argname = NULL) {
 #' Used internally to ensure an object is a list of expression
 #' The output is wrapped in a result, which can contain an ok or
 #' err value.
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/expr__expr.R:132]:
```suggestion
```

File: R/expr__expr.R [hunk 19]
```diff
@@ -575,7 +575,7 @@ Expr_is_not_null = "use_extendr_wrapper"
 #' @param ...  any Expr or string
 #'
 #'
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/expr__expr.R:578]:
```suggestion
```

File: R/rust_result.R [hunk 20]
```diff
@@ -116,7 +116,7 @@ unwrap_or = function(x, or) {
 #' @description DEPRECATED USE stopf instead
 #' @param err error msg string
 #' @param call calling context
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/rust_result.R:119]:
```suggestion
```

File: R/utils.R [hunk 21]
```diff
@@ -177,7 +175,7 @@ pcase = function(..., or_else = NULL) {
 #' @param element_names names of elements to move, if named names, then name of name is to_env name
 #' @param remove bool, actually remove element in from_env
 #' @param to_env env to
-#' @keywords internal
+#' @noRd
```
Comment (eitsupi) [inline R/utils.R:178]:
```suggestion
```

**Review by eitsupi — CHANGES_REQUESTED** [review]

**Review by eitsupi — APPROVED** [review]

**Pull request merged**